polyglot.js

```diff
@@ -2,27 +2,56 @@ class PolyGlot {
+				}
+		}
+
+		static get languages() {
+			switch(game.system.id.toLowerCase()){
+				default:
+					return CONFIG[game.system.id.toUpperCase()].languages;
+				case "wfrp4e":
+					let re = /Language \((.+)\)/i;
+					const pack = game.packs.get("wfrp4e.skills");
+					return pack.getIndex().then( itemList => {
+						let langs = {};
+						for (let item of itemList){
+							let found = item.name.match(re);
+							if(found && found.length > 1 && found[1].length >= 1){
+								langs[found[1]] = found[0];
```

Comment:
You use the entire match rather than the captured group here, so the items in the list would all be "Language (foobar)" instead of just showing "foobar". Should be `found[1]` on the right side